XCUITests/UnitTest.xctestplan

```diff
@@ -9,8 +9,7 @@
     }
   ],
   "defaultOptions" : {
-    "codeCoverage" : false,
-    "testRepetitionMode" : "retryOnFailure"
+    "codeCoverage" : false
```

Comment:
Is there a reason we're changing this?

Reply:
I was curious to see if Unit tests were taking too long due to the retries when a test fails or if that was the cause so that tests stuck when running